File: CS/dxSample_ColoredFormHeaders/MainForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace dxSample_ColoredFormHeaders {
    public partial class MainForm : DevExpress.XtraEditors.XtraForm {
        public MainForm() {
            InitializeComponent();
            InitColorefulForms();
        }
        private static void InitColorefulForms() {
            new ColoredHeaderForm(Color.Magenta).Show();
            new ColoredHeaderForm(Color.Yellow).Show();
            new ColoredHeaderForm(Color.DarkOrchid).Show();
        }
    }
}

File: VB/dxSample_ColoredFormHeaders/MainForm.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace dxSample_ColoredFormHeaders
    Partial Public Class MainForm
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
            InitColorefulForms()
        End Sub
        Private Shared Sub InitColorefulForms()
            CType(New ColoredHeaderForm(Color.Magenta), ColoredHeaderForm).Show()
            CType(New ColoredHeaderForm(Color.Yellow), ColoredHeaderForm).Show()
            CType(New ColoredHeaderForm(Color.DarkOrchid), ColoredHeaderForm).Show()
        End Sub
    End Class
End Namespace
